python/hail/ldMatrix.py

```diff
@@ -43,3 +44,19 @@ def to_local_matrix(self):
 
         j_local_mat = self._jldm.toLocalMatrix()
         return DenseMatrix(j_local_mat.numRows(), j_local_mat.numCols(), list(j_local_mat.toArray()), j_local_mat.isTransposed())
+
+    def write(self, fname):
+        """
+        Writes the LD matrix to a file.
+
+        :param fname: the path to which to write the LD matrix
+        :type fname: str
+        """
+
+        self._jldm.write(fname)
+
+
+    @staticmethod
+    def read(fname):
```

Comment:
I also think that `path` is a param name that's more consistent with our other read/write stuff